doc/flame/collision_detection.md

```diff
@@ -268,22 +268,22 @@ collisions to the whole hat, instead of for just each hitbox separately.
 
 ## Broad phase
 
-If your game field is small and do not have a lot of collidable components - you don't have to
+If your game field isn't huge and does not have a lot of collidable components - you don't have to
 worry about the broad phase system that is used, so if the standard implementation is performant
 enough for you, you probably don't have to read this section.
 
 A broad phase is the first step of collision detection where potential collisions are calculated.
-To calculate these potential collisions are a lot cheaper to calculate than to check the exact
-intersections directly and it removes the need to check all hitboxes against each other and
-therefore avoiding O(n²). The broad phase produces a set of potential collisions (a set of
-`CollisionProspect`s), this set is then used to check the exact intersections between hitboxes, this
-is sometimes called narrow phase.
+Calculating these potential collisions is faster than to checking the intersections exactly, and it removes the need to check all hitboxes against each other and
```

Comment:
There is a 100 char line length limit on the docs, so some of these lines will fail the pipeline

Reply:
I tried to address this throughout the doc with my latest commit.